### DISCUSSION TITLE
Cloudflared Linux build support

### DISCUSSION THREAD

**Discussion by Immortalin** [Homebrew/discussions#4847] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.16
ORIGIN: https://github.com/Homebrew/brew
HEAD: c0c8a4dd4dd19ec16deb429abd50cdf82ea33b27
Last commit: 2 days ago
Core tap JSON: 17 Oct 03:54 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 2.6.10 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: 16-core 64-bit zen
Clang: 17.0.2
Git: 2.25.1 => /bin/git
Curl: 7.68.0 => /bin/curl
Kernel: Linux 5.15.90.1-microsoft-standard-WSL2 x86_64 GNU/Linux
OS: Ubuntu 20.04.6 LTS (focal)
WSL: 2 (Microsoft Store)
Host glibc: 2.31
/usr/bin/gcc: 9.4.0
/usr/bin/ruby: N/A
glibc: 2.35_1
gcc@11: N/A
gcc: 13.2.0
xorg: N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Package request: Cloudflared Linux build

https://github.com/Homebrew/homebrew-core/blob/2e7b6519a3d8105e604356ae30d0396ce525d813/Formula/c/cloudflared.rb

Current formula only supports macOS.

**Comment by ZhongRuoyu** [comment, marked as answer]
> Current formula only supports macOS.

No, `cloudflared` already suports x86_64 Linux and [there's already a bottle available](https://github.com/Homebrew/homebrew-core/blob/2e7b6519a3d8105e604356ae30d0396ce525d813/Formula/c/cloudflared.rb#L18). Are you not able to install it?
